docs/data/base/components/select/UnstyledSelectBasic/css/index.js

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import Select, { selectClasses } from '@mui/base/Select';
+import Option, { optionClasses } from '@mui/base/Option';
+import { useTheme } from '@mui/system';
+
+export default function UnstyledSelectBasic() {
+  return (
+    <React.Fragment>
+      <Select
+        className="CustomSelect"
+        slotProps={{
+          listbox: { className: 'CustomListbox' },
+          popper: { className: 'CustomPopper' },
+        }}
+        defaultValue={10}
+      >
+        <Option className="CustomOption" value={10}>
```

Comment:
```suggestion
        <Option className="CustomSelect-option" value={10}>
```

These classnames makes more sense.